src/Autowhatever.js

```diff
@@ -281,7 +283,7 @@ export default class Autowhatever extends Component {
 
     return (
       <div {...theme(`react-autowhatever-${id}-container`, 'container', isOpen && 'containerOpen')}>
-        <input {...inputProps} />
+        {this.props.renderInput(inputProps)}
```

Comment:
Oh, of course! 😄 

The `inputElement` idea [was proposed in the past](https://github.com/moroshko/react-autowhatever/pull/4). I was against this idea back then, mainly because this means that we cannot promise accessibility out of the box if we don't have full control over the input rendering.

But, I've changed my mind.

Will `inputElement` be enough for your use case, or you need the more flexible `renderInput` prop? I probably prefer the `inputElement` prop at this point.